src/renderer/families/cosmos/operationDetails.js

```diff
@@ -0,0 +1,375 @@
+/* eslint-disable consistent-return */
+// @flow
+import React from "react";
+
+import type { Currency, Unit, Operation, Account } from "@ledgerhq/live-common/lib/types";
+import type {
+  CosmosDelegationInfo,
+  CosmosValidatorItem,
+} from "@ledgerhq/live-common/lib/families/cosmos/types";
+
+import { getDefaultExplorerView, getAddressExplorer } from "@ledgerhq/live-common/lib/explorers";
+import { getAccountCurrency, getAccountUnit } from "@ledgerhq/live-common/lib/account";
+import { formatCurrencyUnit } from "@ledgerhq/live-common/lib/currencies";
+
+import { useCosmosPreloadData } from "@ledgerhq/live-common/lib/families/cosmos/react";
+
+import { openURL } from "~/renderer/linking";
+
+import {
+  OpDetailsTitle,
+  Address,
+  OpDetailsData,
+  OpDetailsVoteData,
+  B,
+} from "~/renderer/modals/OperationDetails/styledComponents";
+import { Trans } from "react-i18next";
+import Box from "~/renderer/components/Box/Box";
+
+import Text from "~/renderer/components/Text";
+import { useDiscreetMode } from "~/renderer/components/Discreet";
+import { BigNumber } from "bignumber.js";
+
+/** @TODO cosmos update this url */
+const helpURL = "https://support.ledger.com/hc/en-us/articles/360013062139";
+
+function getURLFeesInfo(op: Operation): ?string {
+  if (op.fee.gt(200000)) {
```

Comment:
this is weird. it needs (1) code documentation (2) maybe to put the number into a constant? (3) code sharing in live-common?